model_specific/SM/decays/standard_model_decays.hpp

```diff
@@ -0,0 +1,221 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+
+/**
+ * @file standard_model_decays.hpp
+ *
+ * @brief contains class for calculating particle decays
+ *
+ * This file was generated with FlexibleSUSY 2.7.1 and SARAH 4.14.5 .
+ */
+
+#ifndef STANDARDMODEL_DECAYS_H
+#define STANDARDMODEL_DECAYS_H
+
+#include "lowe.h"
+#include "wrappers.hpp"
+#include "error.hpp"
+#include "physical_input.hpp"
+#include "decays/flexibledecay_settings.hpp"
+#include "decays/flexibledecay_problems.hpp"
+
+#include "standard_model.hpp"
+#include "cxx_qft/standard_model_qft.hpp"
+#include "decays/decay_amplitudes.hpp"
+#include "standard_model_decay_table.hpp"
+#include "config.h"
+
+#include <cmath>
+#include <string>
+#include <tuple>
+#include <vector>
+
+namespace flexiblesusy {
+
+class Standard_model_decays {
+public:
+   Standard_model_decays() = default;
+   Standard_model_decays(standard_model::Standard_model model_, softsusy::QedQcd const& qedqcd_,
+         Physical_input const& physical_input_,
+         FlexibleDecay_settings const& flexibledecay_settings_)
+      : model(model_)
+      , qedqcd(qedqcd_)
+      , physical_input(physical_input_)
+      , flexibledecay_settings(flexibledecay_settings_)
+      {}
+   Standard_model_decays(const Standard_model_decays&) = default;
+   Standard_model_decays(Standard_model_decays&&) = default;
+   ~Standard_model_decays() = default;
+   Standard_model_decays& operator=(const Standard_model_decays&) = default;
+   Standard_model_decays& operator=(Standard_model_decays&&) = default;
+
+   const standard_model_decay_table& get_decay_table() const;
+   const FlexibleDecay_problems& get_problems() const;
+
+   void clear();
+   void clear_problems();
+   void calculate_decays();
+
+   const Decays_list& get_hh_decays() const { return decay_table.get_hh_decays();
+      }
+   void calculate_hh_decays();
+
+double partial_width_hh_to_VGVG(standard_model::Standard_model);
```

Comment:
I think this and the following function can be made `const`.

Reply:
See above.